### PR TITLE
Added a more telegram notifications options

### PR DESCRIPTION
**Pull request by tminei**
Tick the checkbox if you understand [x]: 
- [x] I have read and understand the pull request rules.

# Description
Added a more telegram notifications options such as thread id, silent notifications and forward protect.

Note: There is no change in default behavior.

[Thread feature in Telegram.](https://core.telegram.org/bots/api#november-5-2022)

## Type of change

Please delete any options that are not relevant.

- User interface (UI)
- New feature (non-breaking change which adds functionality)

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I ran ESLint and other linters for modified files
- [x] I have performed a self-review of my own code and tested it
- [x] I have commented my code, particularly in hard-to-understand areas
  (including JSDoc for methods)
- [x] My changes generate no new warnings

## Screenshots (if any)
image

### CODE REVIEW

File: src/languages/en.js [hunk 1]
```diff
@@ -216,6 +216,12 @@ export default {
     supportTelegramChatID: "Support Direct Chat / Group / Channel's Chat ID",
     wayToGetTelegramChatID: "You can get your chat ID by sending a message to the bot and going to this URL to view the chat_id:",
     "YOUR BOT TOKEN HERE": "YOUR BOT TOKEN HERE",
+    "Silent Notification": "Silent Notification",
+    "Silent Notification Description": "If enabled, the notification in Telegram will be without sound.",
+    "Protect Forwarding": "Protect Forwarding",
+    "Protect Forwarding Description": "If enabled, the notification in Telegram will be protected from forwarding and saving.",
```
Comment (tminei) [inline src/languages/en.js:222]: Maybe change it to
"Protect Forwarding Description": "If enabled, the notification in Telegram will be protected from forwarding and saving.",

**Comment by furkanipek** [comment]
Great work, exactly what I needed.

**Comment by louislam** [comment]
Ops, I just realized that #2716 is part of this. I will still merge this pr since it got more options.

**Comment by louislam** [comment]
:/ Also found it is duplicate of #2334

Maybe I should merge #2334, since it was submitted early.

I will still give you the credit in the next release notes, since your pr is early than #2716.

**Pull request merged**